core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -718,6 +731,8 @@ public static synchronized void resetAllMetrics() {
       METRIC_REGISTRY.timer(timerName);
     }
     LAST_REPORTED_METRICS.clear();
+    LOG.info("Reset all metrics in the metrics system in {}ms",
+        System.currentTimeMillis() - startTime);
```

Comment:
I feel like this will be a spammy log at `INFO` level

Reply:
`ResetAllMetrics()` is used very often in tests, but should be called really rarely in production. It will only be called when the Master starts and when the `fsadmin metrics clear` is called. Since it may affect the metrics recording and delays some metrics heartbeat, logging is useful.

Reply:
fair enough